enclave_manager/avalon_enclave_manager/wpe/wpe_requester.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import json
+import random
+import logging
+import argparse
+
+import config.config as pconfig
+import utility.logger as plogger
+from http_client.http_jrpc_client import HttpJrpcClient
+
+logger = logging.getLogger(__name__)
+
+
+class WPERequester():
+    """
+    Listener to handle requests from WorkerProcessingEnclave(WPE)
```

Comment:
WorkerProcessingEnclave -> WorkOrderProcessorEnclaveManager .

This class sends the requests from WorkOrderProcessorEnclaveManager to KME Listener